### PR TITLE
u-boot-iot2050: flashing documentation update

### PR DESCRIPTION
**Pull request by fr0st61te**
The flashing documentation update with external flash programmers.
Add the image of the board's back side with flash chip's location.

Signed-off-by: Ivan Mikhaylov <ivan.mikhaylov@siemens.com>

### CODE REVIEW

**Comment by BaochengSu** [comment]
I am not sure if that's a good idea to put this information here, I personally prefer to add it into the [wiki](https://github.com/siemens/meta-iot2050/wiki) and then link here.

File: recipes-bsp/u-boot/README.md [hunk 1]
```diff
@@ -43,3 +43,18 @@ on the host machine:
...
+
+> :warning:
+> This way is not preferred, please, use Dediprog flash programmers if possible.
+
+![overview](back_iot2050.png)
+
+Attach the grabber to the flash chip, then run the following on the host machine:
+
+```shell
+flashrom -p ch341a_spi -c W25Q128.V -w iot2050-image-boot.bin
```
Comment (BaochengSu) [inline recipes-bsp/u-boot/README.md:59]: `apt-get install flashrom` first?

Comment (BaochengSu) [reply recipes-bsp/u-boot/README.md:59]: ...other settings needed?

Comment (BaochengSu) [reply recipes-bsp/u-boot/README.md:59]: Does this programmer reliably support the flash autoprobe?

Comment (fr0st61te) [reply recipes-bsp/u-boot/README.md:59]: > `apt-get install flashrom` first?

Sure, have to add it then.

Comment (fr0st61te) [reply recipes-bsp/u-boot/README.md:59]: > ...other settings needed?

nope

Comment (fr0st61te) [reply recipes-bsp/u-boot/README.md:59]: > Does this programmer reliably support the flash autoprobe?

Yes.

**Review by BaochengSu — APPROVED** [review]

**Pull request merged**